### PR TITLE
Add Innity ad server to amp-ad

### PR DESCRIPTION
**Pull request by engtat**
Add Innity ad server support to amp-ad
Company: Innity

### CODE REVIEW

**Comment by engtat** [comment]
Fixed CLA due to different email address

File: ads/_config.js [hunk 1]
```diff
@@ -374,6 +374,11 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  innity: {
+    prefetch: 'https://cdn.innity.net/admanager.js',
```
Comment (zhouyx) [inline ads/_config.js]: we have `renderStart` and `noContent` [API](https://github.com/ampproject/amphtml/blob/master/ads/README.md#available-apis) that helps to improve user experience. Please consider implementing them.

File: ads/ads.extern.js [hunk 2]
```diff
@@ -291,6 +291,16 @@ _inmobi.getNewAd;
 data.siteid;
 data.slotid;
 
+// innity.js
+var innity_adZone;
+var innityAMPZone;
+var innityAMPTag;
+data.pub;
+data.zone;
+data.width;
```
Comment (zhouyx) [inline ads/ads.extern.js]: data.width data.height exist for every 3p iframe. Please do not add them.

File: ads/innity.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function innity(global, data) {
+  validateData(data, ['pub', 'zone'], ['channel']);
+  loadScript(global, 'https://cdn.innity.net/admanager.js', () => {
+    const innityAMPZone = global.innity_adZone;
+    const innityAMPTag = new innityAMPZone(data.pub, data.zone,
```
Comment (zhouyx) [inline ads/innity.js]: up to your implementation. But do we need `encodeURIComponent` to encode data.pub data.zone data.channel?

File: examples/ads.amp.html [hunk 4]
```diff
@@ -121,6 +121,7 @@
         <option>imedia</option>
         <option>industrybrains</option>
         <option>inmobi</option>
+		<option>innity</option>
```
Comment (zhouyx) [inline examples/ads.amp.html]: nit: please fix indentation.

**Comment by engtat** [comment]
Please review, thanks.

**Comment by zhouyx** [comment]
LGTM. Please follow googlebot [instruction](https://github.com/ampproject/amphtml/pull/9938#issuecomment-308674592) to fix CLA.

**Comment by zhouyx** [comment]
@jridgewell I've already approved the PR.  Could you help with this and merge the PR when CLA is ready while I am OOO 😄  Thanks!

**Comment by engtat** [comment]
@jridgewell @zhouyx For the CLA, not sure anything I have missed, I already have both Individual Agreements and Corporate Agreements ready under my email. Please advise.

**Comment by jridgewell** [comment]
You can rebase updating the authorship:

```js
git config user.email "engtat@innity.com"
git rebase master -exec "git commit --amend --reset-author -CHEAD"
git push -f origin master
```

**Comment by engtat** [comment]
@jridgewell Done.

**Comment by jridgewell** [comment]
Did you force push it?

**Comment by engtat** [comment]
@jridgewell Yes! I forced an update 4 days ago (8b10da5) and all checks restarted. And again, today I did another force push (f6a0592)

**Comment by jridgewell** [comment]
We're still not seeing the rebased commits pushed. What commands did you run?

**Comment by engtat** [comment]
Thanks @jridgewell @zhouyx
looks good now.

**Pull request merged**